lib/src/builders/method.dart

```diff
@@ -186,7 +186,18 @@ abstract class ConstructorBuilder
         HasStatements,
         ValidClassMember {
   /// Create a new [ConstructorBuilder], optionally with a [name].
```

Comment:
Doc comment is no longer complete. This is a case where we can add useful information explaining the 'super' arguments ;)